app/controllers/Authentication.scala

```diff
@@ -254,6 +255,31 @@ class Authentication @Inject() (
     )
   }
 
+  def getToken = SecuredAction.async { implicit request =>
+    for{
+      maybeOldAuthenticator <- env.combinedAuthenticatorService.tokenDao.findByLoginInfo(request.identity.loginInfo)
+      combinedAuthenticator <- env.combinedAuthenticatorService.createToken(request.identity.loginInfo)
+    }yield{
+      var js = Json.obj()
+      if(maybeOldAuthenticator.isDefined){
+        js = Json.obj("token" -> combinedAuthenticator.id, "msg" -> Messages("auth.addedNewToken"))
+      } else {
+        js = Json.obj("token" -> combinedAuthenticator.id)
+      }
+      Ok(js)
+    }
+  }
+
+  def deleteToken = SecuredAction.async { implicit request =>
+    for{
+      maybeOldAuthenticator <- env.combinedAuthenticatorService.tokenDao.findByLoginInfo(request.identity.loginInfo)
+      oldAuthenticator <- maybeOldAuthenticator ?~> Messages("auth.noToken")
+      result <- env.combinedAuthenticatorService.discardToken(oldAuthenticator, Redirect("/dashboard")) //maybe add a way to inform the user that the token was deleted
```

Comment:
what way would that be? how is this used from the outside?
(before ultimately merging this into master, the comment should be removed)

Reply:
You use it by going to '/api/deleteToken'. The result is a Redirect to the dashboard. I think it would be nice to give the user a feedback so he knows that it has been deleted (for example a toast message in the upper right corner)